web/server/codechecker_server/api/report_server.py

```diff
@@ -356,6 +356,13 @@ def process_source_component_filter(session, component_names):
     return or_(*OR)
 
 
+def filter_open_reports_in_tags(results, tag_ids):
+    return results.outerjoin(
+        RunHistory, RunHistory.run_id == Report.run_id) \
+        .filter(RunHistory.id.in_(tag_ids)) \
+        .filter(get_open_reports_date_filter_query())
```

Comment:
We can filter by the run ids parameter too with this function:
```suggestion
def filter_open_reports_in_tags(results, run_ids, tag_ids):
    """ TODO """
    if run_ids:
        results = results.filter(Report.run_id.in_(run_ids))

    if tag_ids:
        results = results \
            .outerjoin(RunHistory,
                            RunHistory.run_id == Report.run_id) \
            .filter(RunHistory.id.in_(tag_ids)) \
            .filter(get_open_reports_date_filter_query())

    return results
```
Also the documentation string is missing.